draco/synthesis/noise.py

```diff
@@ -98,19 +82,14 @@ def process(self, data):
         """
         # Distribute in something other than `stack`
         data.redistribute("freq")
-        # Visibility to be replaced by noise
-        vis = data.vis[:]
 
-        # create a random generator, and create a local seed state
-        rg = RandomGenerator()
-
-        with mpi_random_seed(self.seed, gen=rg) as rg:
-            noise = rg.normal(
-                scale=np.sqrt(tools.invert_no_zero(data.weight[:]) / 2),
-                size=(2,) + data.weight[:].shape,
-            )
-            vis[:] = noise[0] + 1j * noise[1]
+        # Replace visibilities with noise
+        vis = data.vis[:]
+        random.complex_normal(
+            scale=tools.invert_no_zero(data.weight[:]), out=vis, rng=self.rng
```

Comment:
```suggestion
            scale=np.sqrt(tools.invert_no_zero(data.weight[:])), out=vis, rng=self.rng
```
I think there should be a `sqrt` here since `complex_normal` takes the standard deviation.

Reply:
Oooh good catch. Mea cupla.